lib/metadataTypes/definitions/Automation.definition.js

```diff
@@ -329,7 +329,7 @@ module.exports = {
         legacyId: {
             isCreateable: false,
             isUpdateable: false,
-            retrieving: true,
```

Comment:
this is often very useful, sure you wanna remove it?

Reply:
it's removed AFTER postRetrieveTasks --> we can use it to do things (and we are for notifications) but then we don't need to save it